engine/src/main/java/org/destinationsol/game/screens/ShipMixedControl.java

```diff
@@ -38,15 +39,15 @@
     public final SolUiControl shoot2Ctrl;
     public final SolUiControl abilityCtrl;
     private final SolUiControl myDownCtrl;
-    private final Vector2 myMouseWorldPos;
+    private final Vector2 myMouseScreenPos;
```

Comment:
Please no `my` on any new/changed variables. We're slowly trying to get rid of that prefix

Reply:
Fixed in next commit